collection-scripts/gather_hco

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+NS=openshift-cnv
+
+BASE_COLLECTION_PATH="/must-gather"
+
+{
+  PS4='---\n$ '
+  set -x
+  mkdir -p $BASE_COLLECTION_PATH
+  oc get pods --namespace $NS
+  oc get catalogsource --all-namespaces
```

Comment:
or we could add this [here](https://github.com/openshift/must-gather/blob/70c694132bedadaa910cd7d8be877a4478801c37/collection-scripts/gather#L29):
`resources+=(catalogsource)`

Reply:
Understood - I'm just executing the steps the HCO needs.